service/history/execution/context.go

```diff
@@ -351,31 +339,6 @@ func (c *contextImpl) LoadWorkflowExecution(
 		)
 	}
 
-	flushBeforeReady, err := c.mutableState.StartTransaction(domainEntry)
```

Comment:
We no longer need to `StartTransaction` after loading persistence mutable state? I thought it will also update the current version for the workflow?